doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -89,169 +70,245 @@ Workaround: Use the legacy adapter instead of the Elytron adapter. Enter the fol
+. Start the {KIE_SERVER}.
+. Deploy a service with a `SNAPSHOT` version onto the server template.
 
-Workaround: If the sample project that failed to import is present in your space, remove it then import it again.
+Expected result: {CENTRAL} displays a message that the server is in production mode and the service can not be deployed.
 
-ifdef::PAM[]
+Actual result: The deployment starts and fails.
 
-//approved
-.In the guided rule editor, you cannot use the `is contained in comma separated list` constraint in combination with complex values [https://issues.jboss.org/browse/RHPAM-2457[RHPAM-2457]]
+Workaround: Manually delete the server template in {CENTRAL} before restarting the {SERVER}. Alternatively, change the name of the {SERVER} when you reconfigure it.
 
-Issue: In the guided rule editor, you cannot use the `is contained in comma separated list` constraint in combination with complex values. Complex values are values that contain a comma or are wrapped by brackets.
+.When you save a filter and then activate it, an error message displays [https://issues.redhat.com/browse/RHPAM-2545[RHPAM-2545]]
 
-Workaround: None.
+When you save a filter and immediately after this action activate it from the *Saved filters* view, an error message displays.
 
-//approved
-.The *Bulk Reassign* check boxes on the *Task List* page reset before finishing the selected operation [https://issues.jboss.org/browse/RHPAM-2387[RHPAM-2387]]
+Steps to reproduce:
+
+. Open the *Manage Process Instances* perspective.
+. Configure a quick filter.
+. Save the filter.
+. Click the image:ReleaseNotes/maximize-button.png[] *Saved filters* dock icon.
+. Click the new filter to activate it.
+
+Expected result: The filter is activated.
+
+Actual result: An *Unexpected system error* message displays. The filter is activated.
+
+Workaround: none.
 
-Issue: If you select multiple tasks on the *Task List* page and then select a bulk reassignment operation, when the dialog box appears over the *Task List* page, the previously selected tasks are no longer selected.
+.In scenario simulation, the right-hand *Test tools* panel displays the name of a field instead of the type [https://issues.redhat.com/browse/RHDM-1153[RHDM-1153]]
 
-NOTE: This issue is resolved with {PRODUCT} patch release 7.5.1.
+Issue: In come cases, instead of the type of a field, the right-hand *Test tools* panel displays the name of the field.
 
 Steps to reproduce:
 
-* On the *Task List* page, select multiple tasks and choose a bulk reassignment operation.
+. Open an existing DMN file in {CENTRAL}
+. Add a new structure, for example, `Vacation`
+. Add a new field to `Vacation`, for example, `Price` of type `number`.
+. Start editing the Price field. 
+. Click *Add Constraints* and enter some constraints for the `Price` field.
+. Save all changes to the DMN file and close the file.
+. Create a new Test Scenario for the DMN file.
+. Select a column. 
+. Expand the right-hand panel.
+. Click the *Test Tools* dock.
```

Comment:
Is there a difference between the Test Tools doc and the Test tools panel? If they are the same thing, choose doc or panel, and correct capitalization.